src/app/api/crud.py

```diff
@@ -1,35 +1,36 @@
-from typing import Optional, Tuple, Any, List
+from typing import Optional, Tuple, Any, List, Dict
 from app.db import database
 from sqlalchemy import Table
 from pydantic import BaseModel
+from fastapi import HTTPException, Path
 
 
-async def post(payload: BaseModel, table: Table):
+async def post(payload: BaseModel, table: Table) -> int:
     query = table.insert().values(**payload.dict())
     return await database.execute(query=query)
 
 
-async def get(entry_id: int, table: Table):
+async def get(entry_id: int, table: Table) -> Dict[str, Any]:
```

Comment:
I would agree if we had complex types, but this is really basic, so I don't really see how it would help. With the current suggestion, each typing is self-explanatory but if I were to see `JsonDict` I'd have to trace back where it's defined to understand!

But as per your next comment, it would be better to discuss this in another issue :ok_hand: